src/app/app.component.ts

```diff
@@ -75,31 +82,34 @@ export class AppComponent {
 @Component({
   selector: 'app-calendar-header',
   template: `
-    <mat-select placeholder="Calendar Format" [formControl]="form.controls['calendar']">
-      <mat-option *ngFor="let cal of supportedCalendars" [value]="cal">{{cal}}</mat-option>
-    </mat-select>
+        <mat-select placeholder="Calendar" [formControl]="calendar">
```

Comment:
@flavens I tried to wrap the `mat-select` with `mat-form-field` but this led to a CSS problem. In `styles.scss` I added `.mat-form-field-infix` but it did not have an effect. Could you give me a hand here?

Reply:
@tobiasschweizer yes I can this afternoon

Reply:
> @tobiasschweizer yes I can this afternoon

great, thx

Reply:
@tobiasschweizer how can I help? what do you want to do? how can I run it on my machine?

Reply:
I tried to put the whole `mat-select` in a `mat-form-field` 

```html
    <mat-form-field>
        <mat-select placeholder="Calendar" [formControl]="calendar">
          <mat-option *ngFor="let cal of supportedCalendars" [value]="cal">{{cal}}</mat-option>
        </mat-select>
    </mat-form-field>
    <mat-calendar-header></mat-calendar-header>
```

and then the CSS seemed broken. I suppose the styles in `styles.scss`weren't applied any more.

Steps to run this locally:
- check out this branch
- run `npm run build-lib`
- run `npm run ng s`
- access `http://localhost:4200/` and open one of the datepickers

![Screenshot 2020-04-20 at 16 49 05](https://user-images.githubusercontent.com/6000023/79765403-de4ff780-8326-11ea-9445-7ee0c43cdd25.png)

I tried to add `.mat-form-field-infix` to `styles.scss` but it didn't have any effect.

Reply:
I think I just don't understand how scss works ...

Reply:
where did you add this code block?
```
<mat-form-field>
        <mat-select placeholder="Calendar" [formControl]="calendar">
          <mat-option *ngFor="let cal of supportedCalendars" [value]="cal">{{cal}}</mat-option>
        </mat-select>
    </mat-form-field>
    <mat-calendar-header></mat-calendar-header>
```

Reply:
```css
app-calendar-header {
  .mat-select {
    background: #00796b;
    color: white;
    padding: 20px;
    margin-bottom: -20px;
    box-sizing: border-box;
  }

  .mat-select-value,
  .mat-select-arrow {
    color: #fff;
  }
}

.mat-datepicker-content {
  height: 400px;
}
```

I tried to add `.mat-form-field-infix` as a parent selector